app/styles/cart.scss

```diff
@@ -0,0 +1,204 @@
+  strong {
+    font-size: 18px;
+    color: #000;
+    font-weight: 700;
+  }
+  span {
+    font-size: 14px;
+    padding: 10px;
+  }
+}
+.startShopping {
+  width: calc(100% - 20px);
+  text-align: center;
+  padding: 15px;
+  border-radius: 5px;
+  position: absolute;
+  bottom: 10px;
+}
+.cart-with-item {
+  background: #eee;
+  padding-top: 15px;
+  .cart-item-inner {
+    height: 510px;
+    overflow:auto;
+}
+  ul {
+    li {
+      background: #fff;
+      padding:10px;
+      margin-bottom:12px;
+      img,
+      .title-quantity,
+      .product-total-price {
+        display: inline-block;
+        vertical-align: middle;
+      }
+      img {
+        width: 18%;
+      }
+      .title-quantity {
+        width: 67%;
+        padding-left:12px;
+        padding-top:12px
+        h4 {
+          font-size: 18px;
+          font-weight: 700;
+        }
+        .quantity-action {
+          display: inline-block;
+          margin-right: 10%;
+        }
+        .item-price {
+          display: inline-block;
+          font-size: 16px;
+          vertical-align: middle;
+          span{
+            padding-left: 20px;
+          }
+        }
+        .quantity-box {
+          margin-top: 12px;
+        }
+        .minus-quantity,
+        .plus-quantity {
+          display: inline-block;
+          font-size: 36px;
+          width: 25px;
+          height: 25px;
+          border-radius: 50%;
+          background: $btnBgColor;
+          text-align: center;
+          line-height: 19px;
+          color: #fff;
+          padding-left: 1px;
+          position: relative;
+          top: 1px;
+          cursor:pointer;
+        }
+        .plus-quantity {
+          font-size: 22px;
+          line-height: 22px;
+          top: -1px;
+        }
+        input {
+          display: inline-block;
+          width: 45px;
+          padding: 5px;
+          border: 0;
+          text-align: center;
+          line-height: 1;
+          &:disabled {
+            background: #fff;
+          }
+        }
+      }
+      .product-total-price {
+        width: 15%;
+        position: relative;
+        top: 15px;
+      }
+    }
+  }
+}
+
+.lowest-price {
+  background: #fff;
+  margin: 12px;
+  padding: 10px;
+  border-radius: 5px;
+  img ,span{
+    display: inline-block;
+    width: 25%;
+    vertical-align:middle;
+}
+span{
+    width: 75%;
+    font-size: 15px;
+    padding-left: 6%;
+    color:#565656;
+}
+}
+
+
+.cart-footer {
+  background: #fff;
+  padding: 15px 12px;
+  border-top: 1px solid #c7c7c7;
+  & > span{
+    font-size: 15px;
+    display:block;
+    text-align: center;
+    padding: 0px 10px 12px;
+    }
+.checkout {
+  width: 100%;
+  border-radius: 5px;
+  padding: 20px 15px;
+  .cart-total-price, .proceed-checkout {
+    font-size: 18px;
+    display: inline-block;
+    width: 50%;
+    text-align: left;
+}
+.cart-total-price{
+  text-align:right;
+  padding-right:5px;
+  span{
+    padding-left:7px;
+  }
+}
+}
+}
```

Comment:
take care of variables in the entire file.
File seems too big, is this all just for styling cart?

Reply:
Done.
Yes.